src/elements/content-sidebar/__tests__/SidebarNavSign.test.tsx

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
+import { render, fireEvent } from '@testing-library/react';
+import SidebarNavSign from '../SidebarNavSign';
+// @ts-ignore Module is written in Flow
+import FeatureProvider from '../../common/feature-checking/FeatureProvider';
+
+describe('elements/content-sidebar/SidebarNavSign', () => {
+    const onClickRequestSignature = jest.fn();
+    const onClickSignMyself = jest.fn();
+
+    const renderComponent = (props = {}, features = {}) =>
+        render(
+            <FeatureProvider features={features}>
+                <SidebarNavSign {...props} />
+            </FeatureProvider>,
+        );
+    test.each([{ isSignRemoveInterstitialEnabled: true }, { isSignRemoveInterstitialEnabled: false }])(
+        'should render sign button',
+        isRemoveInterstitialEnabled => {
+            const features = {
+                boxSign: {
+                    isSignRemoveInterstitialEnabled: isRemoveInterstitialEnabled,
+                },
+            };
+
+            const wrapper = renderComponent({}, features);
+            expect(wrapper.getByTestId('sign-button')).toBeTruthy();
```

Comment:
I don't have too much experience with react testing-library but what about something like `toBeInTheDocument` or `toBeVisible` instead of `toBeTruthy`? it allows us to be more explicit about what we're looking for although tbh I'm not familiar with the best practice here

Reply:
I agree! it makes sense to use `toBeVisible` here